src/api/services/queries/ledger.ts

```diff
@@ -42,27 +42,29 @@ export function useCreateToken(neighborhood: Network | undefined) {
 
   return useMutation(
     async ({
-      address,
       amount,
+      destination,
       name,
-      symbol,
+      owner,
       precision = 9,
+      symbol,
     }: {
-      address: string;
       amount: string;
+      destination: string;
       name: string;
-      symbol: string;
+      owner: string;
       precision?: number;
+      symbol: string;
```

Comment:
I sorted the arg list so the diff is a little messy, but I'm _removing_ `address` and replacing it with both `owner` and `destination`. The former will be the same as the sender and the latter will take the input address.